tools/nodejs/lib/duckdb.js

```diff
@@ -689,6 +689,14 @@ Statement.prototype.stream;
  */
 Statement.prototype.sql;
 
+/**
+ * @method
+ * @param {...*} params
```

Comment:
These tags aren't accurate? I'm not sure we need to define the full return shape in jsdoc, but at least should give the rough shape

Reply:
Fixed!